mne_bids/stats.py

```diff
@@ -0,0 +1,111 @@
+"""Some functions to extract stats from a BIDS dataset."""
+
+# Authors: Alex Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+from mne_bids import BIDSPath, get_datatypes
+from mne_bids.config import EPHY_ALLOWED_DATATYPES
+
+
+def count_events(root_or_path, datatype='auto'):
+    """Count events present in dataset.
+
```

Comment:
```suggestion

Summarizes events per task denoted by the ``task`` BIDS entity.
```

What if there are no "tasks" enumerated, but there is still an events.tsv file?
e.g.

```
sub-001_ses-01_run-01_ieeg.vhdr
sub-001_ses-01_run-01_events.tsv
...
```?

Reply:
would that be valid BIDS? no task entity?

Reply:
Looks like no: https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/04-intracranial-electroencephalography.html so nvm.

hmm wondering if we should add checks like this eventually into BIDSPath.

If instantiating a BIDSPath that represents a modality-specific file, need certain entities, etc.

Reply:
The only thing I would add, if we don’t have it yet, is a check to ensure we have a task entity when writing